wixcode-users/wix-users.service.json

```diff
@@ -938,14 +937,12 @@
                       "let firstName = // the user's first name",
                       "let lastName = // the user's last name",
                       "let phone =  // the user's phone number",
-                      "let nickname = // the user's nickname",
                       "",
                       "wixUsers.register(email, password, {",
                       "    contactInfo: {",
                       "        \"firstName\": firstName,",
                       "        \"lastName\": lastName,",
-                      "        \"phone\": phone,",
-                      "        \"nickname\": nickname",
+                      "        \"phones\": [phone]",
```

Comment:
The comma is part of the JSON (since it's outside the quotes) but I totally understand why it looks like it isn't. Good potential catch! You can see the example in the branch [here](https://www.wix.com/corvid/reference/wix-users/register?branch=bug2-tov)a